src/main/resources/ApplicationInsights.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ApplicationInsights xmlns="http://schemas.microsoft.com/ApplicationInsights/2013/Settings" schemaVersion="2014-05-30">
+ 
+  <!-- The key from the Azure portal: -->
+  <InstrumentationKey>c2435fd3-cb59-423c-846b-04898b101cc1</InstrumentationKey>
```

Comment:
Security by obscurity. In that way, the key is not in the source, but in TravisCI environment variables, where only JabRef developers have access to by using "echo $key" in the travis.yml in a pull request. If we can somehow encrypt the JAR file, we can further prevent others from stealing the key.